Tests/UITests/TestApp/OnboardingTestsView.swift

```diff
@@ -15,42 +15,41 @@ struct OnboardingTestsView: View {
     @Binding var onboardingFlowComplete: Bool
     @State var showConditionalView = false
 
-
```

Comment:
Nit: We usually use two new lines between properties and computed properties, we should keep it that way.